content/changelog/2025/03-07-clever-tools-3.12.md

```diff
@@ -0,0 +1,104 @@
+---
+title: 'Clever Tools 3.12, with API Tokens and Network Groups'
+date: 2025-03-07
+tags:
+  - clever-tools
+  - cli
+authors:
+  - name: David Legrand
+    link: https://github.com/davlgd
+    image: https://github.com/davlgd.png?size=40
+  - name: Hubert Sablonnière
+    link: https://github.com/hsablonniere
+    image: https://github.com/hsablonniere.png?size=40
+description: Some tiny changes, before the huge ones
+excludeSearch: true
+---
+
+[Clever Tools 3.12](https://github.com/CleverCloud/clever-tools/releases/tag/3.12.0) is available. It includes some bug fixes and two new experimental features, available through feature flags.
+
+## API Tokens
+First is `clever tokens`, which allows you to create and manage API tokens, and use them to request the Clever Cloud API:
+
+```bash
+clever features enable tokens
```

Comment:
question: can we put the `enable` on its own line (like for NGs) or own code block?